tardis/transport/montecarlo/single_packet_loop.py

```diff
@@ -85,6 +86,9 @@ def single_packet_loop(
     )
 
     rpacket_tracker.track(r_packet)
+    rpacket_tracker.track_boundary_interaction(
+        PacketEjectaStatus.OUT_EJECTA, r_packet.current_shell_id
```

Comment:
I think we should rename this @andrewfullard

Reply:
I would personally just add to the PacketStatus Enum rather than defining a new one